operator/pkg/ciliumendpointslice/manager.go

```diff
@@ -333,10 +336,9 @@ func (c *cesMgr) RemoveCEPFromCache(cepName string, baseDelay time.Duration) {
 	if cesName, exists := c.desiredCESs.getCESName(cepName); exists {
 		var ces *cesTracker
 		if ces, exists = c.desiredCESs.getCESTracker(cesName); !exists {
-			log.Info("Attempted to remove non-existent CES, skipping.")
+			log.Infof("Attempted to remove non-existent cesName :%v , skipping.", cesName)
```

Comment:
Thanks, this kind of info can help debuggability 👍 however, please use structured logging for log messages rather than using the `log.Infof()` varieties. This helps logging systems to aggregate logs appropriately.
```suggestion
			log.WithFields(logrus.Fields{
			    "CiliumEndpointSlice": cesName,
			}.Info("Attempted to remove non-existent CES, skipping.")
```

Reply:
(Same feedback will apply to the other logging messages)